docs/modules/shapes/assets/js/frontend/handlers/text-path.md

```diff
@@ -0,0 +1,43 @@
+# Text-Path.js
+
+## Product Knowledge Base:
+
+- [Text Path on Elementor.com](https://elementor.com/help/text-path-widget/)
```

Comment:
Please add links to Elementor docs & videos of this feature.

Reply:
Done.